File: Coding/Set3/SwaptwoArrays.js
let arr1 = [1,2,3,4,5]
let arr2 = [6,7,8,9,10]

swaparray(arr1, arr2)
console.log(arr1)
console.log(arr2)

function swaparray(arr1, arr2){
    if(arr1.length != arr2.length)
    {
        console.log("arrays cannot be swapped")
        return
    }
    for(let i=0;i<arr1.length;i++){
        let temp = arr1[i]
        arr1[i] = arr2[i]
        arr2[i] = temp
    }
}

File: Coding/Java/Set3/SwapTwoarrays.java
package Set3;

import java.util.Arrays;

public class SwapTwoarrays {
    public static void swaparrays(int[] arr1, int[] arr2){
        if(arr1.length != arr2.length){
            System.out.println("Arrays cannot be swapped");
            return;
        }
        for(int index = 0;index < arr1.length;index++){
            int temp = arr1[index];
            arr1[index] = arr2[index];
            arr2[index] = temp;
        }
    }
    public static void main(String[] args) {
        int[] arr1 = {1,2,3,4,5};
        int[] arr2 = {6,7,8,9,10};

        swaparrays(arr1, arr2);
        System.out.println(Arrays.toString(arr1));
        System.out.println(Arrays.toString(arr2));
    }
}
